recipes/cmsip/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cmsip" %}
+{% set version = "0.0.0.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "c7617ad363d0f07e7689175e21d5af7208d8d81fd8ef4cea65369c922d11bc82"
+
+build:
+  number: 0
+  entry_points:
+    - cmsip=cmsip.cmsip:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3
+  run:
+    - matplotlib
```

Comment:
Doesn't this require samtools and other things to run?

Reply:
Hi Devon,

The run time requirements are added, including samtools, bedtools, R, and bash. Can you help review and merge my PR? Thank you.